src/components/KInput/KInput.vue

```diff
@@ -1,7 +1,8 @@
 <template>
+  <!-- TODO: [beta] change wrapper class to k-input -->
   <div
     class="k-input-wrapper"
-    :class="[$attrs.class, { 'has-error' : charLimitExceeded || hasError }]"
+    :class="[$attrs.class, { 'error' : charLimitExceeded || error || hasError }]"
```

Comment:
suggestion: instead of just using a class of `error` (which is extremely broad) can we at least use `input-error` to be a tad more specific? This follows the pattern you used for classes like `.k-dropdown > .dropdown-menu`

Reply:
Done